x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/analysis/analyzer/Verifier.java

```diff
@@ -282,14 +284,13 @@ private static Failure fail(Node<?> source, String message, Object... args) {
      */
     private static boolean checkGroupBy(LogicalPlan p, Set<Failure> localFailures,
             Map<String, Function> resolvedFunctions, Set<LogicalPlan> groupingFailures) {
-        return checkGroupByAgg(p, localFailures, groupingFailures, resolvedFunctions)
-                && checkGroupByOrder(p, localFailures, groupingFailures, resolvedFunctions)
+        return checkGroupByAgg(p, localFailures, resolvedFunctions)
```

Comment:
I assume the removed parameters were not needed inside the methods?

Reply:
yep